app/web/src/workers/webworker.ts

```diff
@@ -1376,6 +1383,12 @@ const mjolnirJob = async (
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   let req: undefined | AxiosResponse<IndexObjectMeta, any>;
 
+  const sdf = sdfClients[workspaceId];
+  if (!sdf) {
+    debug("no sdf client found for", workspaceId);
```

Comment:
we should `error` this as well as instrumenting it in the span so we can alert on it